plugin/storage/grpc/proto/storage.proto

```diff
@@ -22,7 +22,7 @@ option (gogoproto.sizer_all) = true;
 option (gogoproto.goproto_registration) = true;
 
 message GetDependenciesRequest {
-    google.protobuf.Timestamp end_timestamp = 1 [
+    google.protobuf.Timestamp end_time = 1 [
```

Comment:
I meant replace `lookback` with `start_time` (and swap the order). The Query Service API in the future will be converted to similar gRPC API, but it will need to work over JSON. It's less cognitive load to have two timestamp-formatted fields in JSON than to have a lookback in nanoseconds.